packages/styles/table.css

```diff
@@ -110,11 +110,34 @@
   color: var(--white);
 }
 
-.Table--border,
+.Table--border {
+  border: 1px solid var(--gray-40);
+}
+
 .Table--border .TableHeader,
 .Table--border .TableFooter,
 .Table--border .TableCell {
-  border: 1px solid var(--gray-40);
+  border: 2px solid var(--gray-40);
```

Comment:
The 2px border should only apply to the border between whatever is the header/body. The border between individual cells should still be 1px.